pkg/muxer/muxer.go

```diff
@@ -73,34 +72,45 @@ func (m *Muxer) Stream(samples [][2]float64) (n int, ok bool) {
 		samples[i] = [2]float64{}
 	}
 
+	n = 0
+
 	for m.Len() > 0 && n < toStream {
 		m.Queues.Range(func(key interface{}, value interface{}) bool {
 			st := value.(beep.Streamer)
+
 			id := userid.ID(key.(uint64))
-			// mix the stream
-			sn, sok := st.Stream(tmp[streamedCount[id]:toStream])
-			for i := range tmp[:sn] {
-				samples[i][0] += tmp[i][0]
-				samples[i][1] += tmp[i][1]
-			}
 
-			if !sok {
-				// remove drained streamer
-				m.Delete(id)
+			_, bok := streamedCount[id]
+			if !bok {
+				streamedCount[id] = 0
 			}
+			// mix the stream
+			sn, sok := st.Stream(tmp[streamedCount[id]:toStream])
 
 			streamedCount[id] += sn
 
 			if streamedCount[id] > n {
 				n = streamedCount[id]
 			}
+
+			if !sok {
+				m.Delete(id)
+				return false
```

Comment:
returning false here stops muxing all of the data doesn't it? Wouldn't this cause there to be a loss of data if one person drops out?

Reply:
I think this return is also causing the tests to fail

Reply:
This return false is causing issues on audio playback from other connections